src/plugins/intel_cpu/src/config.cpp

```diff
@@ -231,28 +231,36 @@ void Config::readDebugCapsProperties() {
 
     const char* envVarValue = nullptr;
 
-    if (envVarValue = readEnv("OV_CPU_EXEC_GRAPH_PATH"))
+    envVarValue = readEnv("OV_CPU_EXEC_GRAPH_PATH");
```

Comment:
Just for my education: what exact compilation error you have on Macos?

Reply:
Error (compiler warning as error):
```cpp
openvino_ai/src/plugins/intel_cpu/src/config.cpp:234:21: error: using the result of an assignment as a condition without parentheses [-Werror,-Wparentheses]
    if (envVarValue = readEnv("OV_CPU_EXEC_GRAPH_PATH"))
```